benchmarks/inference/llm-bench/src/llm_bench/benchmark_runner.py

```diff
@@ -0,0 +1,316 @@
+import itertools
+import json
+import multiprocessing
+import os
+import queue
+import sys
+import threading
+import time
+import yaml
+from pathlib import Path
+from typing import List, Optional, Iterable, Tuple, Dict
+
+from loguru import logger
+from pydantic import Field
+from tqdm import tqdm
```

Comment:
Nit: is this `tqdm` import a new library additional to the benchmarking code? If so, perhaps we can any new dependencies to a `requirements.txt` file?